features/issue/types/Issue.ts

```diff
@@ -1,17 +1,28 @@
+import { UserIssues } from '@/shared/types/UserIssues'
+
 export type Issue = {
 	id: number
 	title: string
 	description: string
 	status: Status
 	createdAt: Date
 	updatedAt: Date
+	users: UserIssues[]
+	creatorId: string
 }
-export type IssueFormData = {
+export type NewIssueFormData = {
```

Comment:
CreateIssueFormData.

A nie możesz tych typów ze schemy zodowej wyciągnąć?

type CreateIssueFormData = z.infer<typeof CreateIssueSchema>

https://medium.com/@thiwankajayasiri/typescript-has-become-an-industry-standard-for-writing-robust-javascript-applications-83e3744a050e